projects/common/src/time/time-duration.ts

```diff
@@ -1,13 +1,16 @@
 import { parse, toSeconds } from 'iso8601-duration';
+import { toString } from 'lodash-es';
 import { assertUnreachable } from '../utilities/lang/lang-utils';
 import { TimeUnit } from './time-unit.type';
 
 export class TimeDuration {
   private readonly millis: number;
+  private readonly militaryTime: Date;
```

Comment:
please don't use the term "militaryTime" - it's not universally understood. Further, even though I think I know what it means, I'm not sure why we'd have both millis and this.